system/include/emscripten/threading.h

```diff
@@ -310,6 +310,11 @@ int emscripten_sync_run_in_main_runtime_thread_(EM_FUNC_SIGNATURE sig, void *fun
 //  - Note that multiple asynchronous commands from a single pthread/Worker are
 //    guaranteed to be executed on the main thread in the program order they
 //    were called in.
+//  - Note that synchronous and asynchronous commands are *also* guaranteed to
+//    be executed in order. That is, sync and async commands cannot be
+//    reordered. The only difference between a sync and async command is that
```

Comment:
Maybe instead `That is, sync and async commands will not get reordered in the queue, but will be executed in a sequentially consistent order in which they were dispatched.`?, because "cannot be reordered" can be read a bit vaguely: Who cannot reorder them? Can I not reorder sync and async commands? Why not?

Reply:
Sounds good, thanks, updated.